tests/scripts/test_format_lintro_pr_comment.py

```
@@ -0,0 +1,241 @@
+            "2026-04-17T06:20:00Z	stdout	2026-04-17T06:20:00Z	✨  Running black (check)    🖤 🖤 🖤 🖤 🖤",
+            (
+                "2026-04-17T06:20:05Z	stderr	2026-04-17T06:20:05Z	"
+                "##[error]The runner has received a shutdown signal. This can happen "
+                "when the runner service is stopped, or a manually started runner is canceled."
+            ),
+            (
+                "2026-04-17T06:20:05Z	stderr	2026-04-17T06:20:05Z	"
+                "##[error]Process completed with exit code 143."
+            ),
+        ],
+    )
```

Ruff (0.15.10)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)